sql/src/test/java/org/apache/druid/sql/calcite/util/CalciteTests.java

```diff
@@ -97,6 +97,7 @@
   public static final String DATASOURCE3 = "numfoo";
   public static final String DATASOURCE4 = "foo4";
   public static final String DATASOURCE5 = "lotsocolumns";
+  public static final String DATASOURCE6 = "unnestnumfoo";
```

Comment:
Better name? `nested` perhaps? Also, would be cool to add a comment with the schema: I find it hard to suss that out from the code.